help/Makefile.am

```diff
@@ -17,10 +17,12 @@ SYMLINKS=	topic accept cmode admin names links away whowas \
 
 all: index
 build:
-clean:
 depend:
 lint:
 
+clean:
```

Comment:
We're adding a new `PHONY` target and there are a bunch of other PHONY targets. Normally I'd expect

```
.PHONY: all build clean depend lint index
```

Opinion?

Reply:
We're not adding it, just moving it down a few lines. I can't remember how but some of the autostuff generates a .PHONY in the Makefile proper.